[2022-11-02T09:46:44.746+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_pipe.download_data scheduled__2022-11-01T00:00:00+00:00 [queued]>
[2022-11-02T09:46:44.848+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_pipe.download_data scheduled__2022-11-01T00:00:00+00:00 [queued]>
[2022-11-02T09:46:44.857+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T09:46:44.862+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2022-11-02T09:46:44.872+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T09:46:44.936+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): download_data> on 2022-11-01 00:00:00+00:00
[2022-11-02T09:46:44.945+0000] {standard_task_runner.py:55} INFO - Started process 1297 to run task
[2022-11-02T09:46:44.975+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'covid_pipe', 'download_data', 'scheduled__2022-11-01T00:00:00+00:00', '--job-id', '742', '--raw', '--subdir', 'DAGS_FOLDER/covid_dag.py', '--cfg-path', '/tmp/tmperr7obgu']
[2022-11-02T09:46:44.980+0000] {standard_task_runner.py:83} INFO - Job 742: Subtask download_data
[2022-11-02T09:46:45.516+0000] {task_command.py:376} INFO - Running <TaskInstance: covid_pipe.download_data scheduled__2022-11-01T00:00:00+00:00 [running]> on host 53c1f4b2cdcf
[2022-11-02T09:46:47.013+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=covid_pipe
AIRFLOW_CTX_TASK_ID=download_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-01T00:00:00+00:00
[2022-11-02T09:46:47.060+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-02T09:46:47.073+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl -sSL https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_daily_reports/01-01-2021.csv > /opt/***/01-01-2021.csv']
[2022-11-02T09:46:47.139+0000] {subprocess.py:86} INFO - Output:
[2022-11-02T09:46:49.750+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-11-02T09:46:49.861+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=covid_pipe, task_id=download_data, execution_date=20221101T000000, start_date=20221102T094644, end_date=20221102T094649
[2022-11-02T09:46:49.912+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-02T09:46:49.992+0000] {dagrun.py:683} ERROR - Failed to get task for ti <TaskInstance: covid_pipe.removedata scheduled__2022-11-01T00:00:00+00:00 [None]>. Marking it as removed.
[2022-11-02T09:46:50.011+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
